controllers/names.go

```diff
@@ -50,6 +50,10 @@ func nameGraphBuilderService(instance *cv1.UpdateService) string {
 	return instance.Name + "-graph-builder"
 }
 
+func nameGraphDataPod(instance *cv1.UpdateService) string {
+	return instance.Name + "-graph-data-init-sha"
```

Comment:
can this be `-tag-digest` or something, to avoid being longer than the 14-character `-policy-engine` or `-graph-builder`?  I know we've had name-is-too-long issues in the past, and I'm not clear on what happens if the name returned by these helpers ends up growing long.

Reply:
Also, handling this per-instance means that  if we have multiple UpdateService sharing the same graph-data pullspec, we'll be launching redundant Pods to figure out that digest.  In practice, maybe that situation is rare enough that we don't have to worry about trying to centralize tag-to-digest lookup at the operator level where results can be shared between multiple instances.